docs/source/raft_bft_migration.md

```diff
@@ -0,0 +1,152 @@
+# Migrating from Raft to BFT
+
+Nodes at v3.0.0 or higher allow for users who want to transition channels from using Raft-based ordering services to BFT-based ordering services.
+This can be accomplished through a series of configuration update transactions on each channel in the network.
+
+This tutorial will describe the migration process at a high level, calling out specific details where necessary.
+
+
+## Assumptions and considerations
+Before attempting migration, take the following into account:
+
+1. This process is solely for migration from Raft to BFT. Migrating between any other orderer consensus types is not supported.
+2. Migration is one way. Once the ordering service is migrated to BFT, and starts committing transactions, it is not possible to go back to Raft.
+3. Because the ordering nodes must go down and be brought back up, downtime must be allowed during the migration.
+4. Recovering from a botched migration is possible only if a backup is taken at the point in migration prescribed later in this document. If you do not take a backup, and migration fails, you will not be able to recover your previous state.
+5. All channels must be migrated during the same maintenance window. It is not possible to migrate only some channels before resuming operations.
+6. At the end of the migration process, every channel will have the same consenter set of BFT nodes.
+7. Migration is done in place, utilizing the existing ledgers for the deployed ordering nodes. Addition or removal of orderers should be performed after the migration. 
+
+
+## Migration flow
+Migration is carried out in five phases.
+
+1. The system is placed into a maintenance mode where application transactions are rejected and only ordering service admins can make changes to the channel configuration.
+2. The system is stopped, and a backup is taken in case an error occurs during migration.
```

Comment:
Backup should be done as the first step?

Reply:
No.
In maintenance mode data tx's are blocked, that's how we ensure that we take a backup of a state which is coherent. I will add a comment to clarify this.